frontend/src/components/__tests__/Filter.js

```diff
@@ -128,6 +128,8 @@ describe('filter', () => {
     let topic = await screen.findByRole('combobox', { name: 'topic' });
     userEvent.selectOptions(topic, 'startDate');
 
+    screen.logTestingPlaygroundURL();
```

Comment:
TIL testing-playground. You should give a small demo sometime, I'm interested in how it works. I could be misunderstanding but seems like we can remove this line.

Reply:
whoops yes -  maybe at the engineering meeting